src/features/wallet/root/BalanceCard/BalanceCard.tsx

```diff
@@ -83,7 +83,11 @@ const BalanceCard = ({ onReceivePress, onSendPress }: Props) => {
           />
           <CurrencyBadge
             variant="hst"
-            amount={account?.secBalance?.integerBalance || 0}
+            amount={
```

Comment:
Not sure what was wrong here, but maybe https://github.com/helium/helium-js/pull/146 would help too

Reply:
basically we needed the float balance instead of the integer balance. I just updated to a better way to do that